.env

```diff
@@ -28,7 +28,7 @@ PYTHON_VERSION=python3.7
 
 #### MirrorNode Prefixes & Tags ####
 MIRROR_IMAGE_PREFIX=gcr.io/mirrornode/
-MIRROR_IMAGE_TAG=0.75.0-rc3
+MIRROR_IMAGE_TAG=0.76.0
```

Comment:
Could the `NETWORK_NODE_IMAGE_TAG` image tag be updated to the latest i.e. `0.35.1` as it has the change that prevents delegatecalls to the precompile?

Reply:
We don't have a image for this tag yet. I will try to create one and update all network configs in another PR.